builtin/common/misc_helpers.lua

```diff
@@ -254,6 +254,11 @@ function math.factorial(x)
 	return v
 end
 
+--------------------------------------------------------------------------------
+function math.clamp(v, min, max)
+        return v < min and min or v > max and max or v
```

Comment:
Don't you usually want clamping to be `<=` / `>=`?

Reply:
I'm checking if value less than min, then return min. Meaning min is the minimum allowed value. Anything below min clamps to min. If I changed the comparison to <= and >= then the result would be the same, it would just short circuit sooner. I can still change it if you want.